packages/devtools_extensions/README.md

```diff
@@ -106,9 +106,9 @@ to interact with DevTools. From anywhere your extension web app, you can access
 #### Utilize helper packages
 
 Use [package:devtools_app_shared](https://pub.dev/packages/devtools_app_shared) for access to
-service managers, common widgets, DevTools theming, utilities, and more.
-
-TODO(kenz): link examples.
+service managers, common widgets, DevTools theming, utilities, and more. See the
+[devtools_app_shared/example](https://github.com/flutter/devtools/tree/master/packages/devtools_app_shared/example)
```

Comment:
Ubernit: I think this can just be `See [devtools_app_shared/example](...) for sample usages ...`